adafruit_mpr121.py

```diff
@@ -30,8 +30,26 @@
 * Adafruit's Bus Device library: https://github.com/adafruit/Adafruit_CircuitPython_BusDevice
 """
 
+from __future__ import annotations
+
 import time
 
+try:
+    from typing import List, Optional, Tuple
+except ImportError:
+    # typing module does not exist in CircuitPython
+    class List(list):
+        """Typing hint indicating a list."""
+
+    # pylint: disable=too-few-public-methods
+    class Optional:
+        """Typing hint alias of ``Union[Any, None]``."""
+
+    class Tuple(tuple):
+        """Typing hint indicating a tuple."""
+
+
+import busio
```

Comment:
the busio import can go up inside the try block like this:
```
try:
    from typing import List, Optional, Tuple
    import busio
except ImportError:
    pass
```
This way it won't actually get imported when it runs on a microcontroller since it's only used for typing it will save a bit of RAM by not importing it when it's not needed for execution